quill-jdbc/src/main/scala/io/getquill/sources/jdbc/ConfigJdbcSource.scala

```diff
@@ -0,0 +1,21 @@
+package io.getquill.sources.jdbc
+
+import java.sql.Connection
+
+import io.getquill.JdbcSourceConfig
+import io.getquill.naming.NamingStrategy
+import io.getquill.sources.sql.idiom.SqlIdiom
+
+import scala.util.DynamicVariable
+
+class ConfigJdbcSource[D <: SqlIdiom, N <: NamingStrategy](config: JdbcSourceConfig[D, N])
```

Comment:
Just because of `dataSource`. Are you suggesting to only pass it and remove the `JdbcSourceConfig` here?